restli-server/src/test/java/com/linkedin/restli/server/test/TestRestLiDefaultInResponse.java

```diff
@@ -0,0 +1,126 @@
+/*
+   Copyright (c) 2020 LinkedIn Corp.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+package com.linkedin.restli.server.test;
+
+import com.google.common.collect.ImmutableMap;
+import com.linkedin.data.DataList;
+import com.linkedin.data.DataMap;
+import com.linkedin.data.schema.DataSchema;
+import com.linkedin.data.schema.NamedDataSchema;
+import com.linkedin.data.schema.RecordDataSchema;
+import com.linkedin.data.schema.SchemaFormatType;
+import com.linkedin.data.schema.SchemaParserFactory;
+import com.linkedin.data.schema.generator.AbstractGenerator;
+import com.linkedin.data.schema.resolver.FileDataSchemaResolver;
+import com.linkedin.data.schema.resolver.MultiFormatDataSchemaResolver;
+import com.linkedin.data.template.DataTemplateUtil;
+import com.linkedin.restli.internal.server.response.ResponseUtils;
+import java.io.File;
+import java.io.IOException;
+import org.testng.Assert;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+import org.testng.reporters.Files;
+
+
+public class TestRestLiDefaultInResponse
```

Comment:
Consider renaming this to just be `TestResponseUtils` to be aligned with the class it's testing.